src/migrations/01-create-user.js

```diff
@@ -32,6 +32,9 @@ module.exports = {
     profileImage: {
       type: Sequelize.STRING,
     },
+    role: {
+      type: Sequelize.STRING
+    },
```

Comment:
why didn't you set a default role for a new user being created?. Also add the `allowNull:false` atrribute.